--- yumex-1.9.6/src/views.py~	2007-03-19 15:41:09.000000000 +0100
+++ yumex-1.9.6/src/views.py	2007-05-24 16:05:33.525779332 +0200
@@ -193,14 +193,14 @@
             action = pkg.queued
             if action:            
                 if action in ( 'u', 'i' ):
-                    icon = gtk.STOCK_APPLY
+                    icon = 'network-server'
                 else:
-                    icon = gtk.STOCK_CANCEL
+                    icon = 'edit-delete'
                 cell.set_property( 'visible', True )
-                cell.set_property( 'stock-id', icon )
+                cell.set_property( 'icon-name', icon )
             else:
                 cell.set_property( 'visible', pkg.recent )
-                cell.set_property( 'stock-id', gtk.STOCK_NEW )
+                cell.set_property( 'icon-name', 'document-new' )
         else:
             cell.set_property( 'visible', False )
             
@@ -613,4 +613,4 @@
             fp = open( "/etc/yum/pluginconf.d/%s.conf" % row[1], "wt" )
             row[2].write( fp )
             fp.close()
-            
\ Brak znaku nowej linii na końcu pliku
+            
