--- yumex-1.9.6/misc/yumex.desktop~	2006-09-07 12:33:59.000000000 +0200
+++ yumex-1.9.6/misc/yumex.desktop	2007-05-23 19:32:07.209885889 +0200
@@ -5,7 +5,7 @@
 Comment[zh_TW]=Yum 的圖形使用者介面前端程式
 Categories=Application;System;X-Red-Hat-Base;X-Fedora;
 Icon=/usr/share/pixmaps/yumex/yumex-icon.png
-Exec=/usr/bin/yumex
+Exec=/usr/bin/gnomesu -c /usr/bin/yumex -t "Yum Extender"
 Type=Application
 Terminal=false
 Encoding=UTF-8
