--- yumex-1.9.6/src/packages.py~	2007-03-20 13:00:23.000000000 +0100
+++ yumex-1.9.6/src/packages.py	2007-05-28 11:10:45.761489850 +0200
@@ -172,6 +172,7 @@
                     yp.action = 'i'
                     yield yp
         elif mask == 'updates':
+            self.up.doObsoletes()
             obsoletes = self.up.getObsoletesTuples( newest=1 )
             for ( obsoleting, installed ) in obsoletes:
                 obsoleting_pkg = self.getPackageObject( obsoleting )
--- yumex-1.9.6/src/yumapi.py~	2007-05-30 11:17:44.000000000 +0200
+++ yumex-1.9.6/src/yumapi.py	2007-05-30 11:26:21.570229473 +0200
@@ -165,7 +165,7 @@
                 tx = self.update()
             else:
                 self.yumex_logger.info( _( "--> Preparing for a partial update" ) )
-                #if self.settings.fullobsoletion:
-                self.yumex_logger.info( _( "--> Adding all obsoletion to transaction" ) )
-                self._doFullObsoletion()
+                if self.settings.fullobsoletion:
+                    self.yumex_logger.info( _( "--> Adding all obsoletion to transaction" ) )
+                    self._doFullObsoletion()
                 for po in pkgs['u']:
                    tx = self.update( po.pkg )
                    print tx
